packages/palette/package.json

```diff
@@ -116,6 +116,10 @@
     "@artsy/palette-tokens": "^2.6.0",
     "@seznam/compose-react-refs": "^1.0.6",
     "@styled-system/theme-get": "^5.1.2",
+    "lodash.intersection": "^4.4.0",
+    "lodash.orderby": "^4.6.0",
+    "lodash.reject": "^4.6.0",
+    "lodash.uniqby": "^4.7.0",
```

Comment:
@dzucconi - thoughts on this? I could go into these packages and copy in the code, or i can just pluck the modules. This seems reasonable. Unlike some I really don't like like writing stdlib-esque code from scratch 😄

Reply:
I guess it's fine though it's still adding like ~45kb.

Reply:
Though maybe they have some shared deps and they don't all just add up? (like each of those are exactly 12.3kb)

Reply:
yeah, i bet all the primitives underneath are shared